src/engraving/draw/fontmetrics.h

```diff
@@ -24,7 +24,7 @@
 
 #include "font.h"
 
-#include "modularity/ioc.h"
+#include "framework/global/modularity/ioc.h"
```

Comment:
also avoid need for src/framework/global include directory when including text.h etc.